src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts

```diff
@@ -211,10 +211,10 @@ export const startEntries: GettingStartedStartEntryContent = [
 const Button = (title: string, href: string) => `[${title}](${href})`;
 
 const CopilotStepTitle = localize('gettingStarted.copilotSetup.title', "Use AI features with Copilot for free");
-const CopilotDescription = localize({ key: 'gettingStarted.copilotSetup.description', comment: ['{Locked="["}', '{Locked="]({0})"}'] }, "Write code faster and smarter with [Copilot]({0}) for free with your GitHub account.", product.defaultChatAgent?.documentationUrl ?? '');
+const CopilotDescription = localize({ key: 'gettingStarted.copilotSetup.description', comment: ['{Locked="["}', '{Locked="]({0})"}'] }, "Write code faster and smarter using [Copilot]({0}). With your GitHub account, get 2,000 code completions and 50 chat messages per month for free.", product.defaultChatAgent?.documentationUrl ?? '');
 const CopilotTermsString = localize({ key: 'copilotTerms', comment: ['{Locked="["}', '{Locked="]({0})"}', '{Locked="]({1})"}'] }, "By continuing, you agree to Copilot [Terms]({0}) and [Privacy Policy]({1}).", product.defaultChatAgent?.termsStatementUrl ?? '', product.defaultChatAgent?.privacyStatementUrl ?? '');
-const CopilotSignedOutButton = Button(localize('setupCopilotButton.signIn', "Sign in to use Copilot"), `command:workbench.action.chat.triggerSetup?${encodeURIComponent(JSON.stringify([true]))}`);
-const CopilotSignedInButton = Button(localize('setupCopilotButton.setup', "Setup Copilot"), `command:workbench.action.chat.triggerSetup?${encodeURIComponent(JSON.stringify([true]))}`);
+const CopilotSignedOutButton = Button(localize('setupCopilotButton.signIn', "Sign in to Use Copilot"), `command:workbench.action.chat.triggerSetup?${encodeURIComponent(JSON.stringify([true]))}`);
+const CopilotSignedInButton = Button(localize('setupCopilotButton.setup', "Set Up Copilot"), `command:workbench.action.chat.triggerSetup?${encodeURIComponent(JSON.stringify([true]))}`);
```

Comment:
Do you need to change the first arg so that the string will be relocalized?

Reply:
I don't think any of these have been localized yet since this string is new